src/core/hle/service/am/applets/applet_mii.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright 2022 yuzu Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include "common/assert.h"
+#include "common/logging/log.h"
+#include "core/core.h"
+#include "core/frontend/applets/mii.h"
+#include "core/hle/service/am/am.h"
+#include "core/hle/service/am/applets/applet_mii.h"
+#include "core/reporter.h"
+
+namespace Service::AM::Applets {
+
+Mii::Mii(Core::System& system_, LibraryAppletMode applet_mode_,
+         const Core::Frontend::MiiApplet& frontend_)
+    : Applet{system_, applet_mode_}, frontend{frontend_}, system{system_} {}
+
+Mii::~Mii() = default;
+
+void Mii::Initialize() {
+    complete = false;
+
+    const auto storage = broker.PopNormalDataToApplet();
+    ASSERT(storage != nullptr);
+
+    const auto data = storage->GetData();
+    ASSERT(data.size() == sizeof(MiiAppletInput));
+
+    std::memcpy(&input_data, data.data(), sizeof(MiiAppletInput));
+}
+
+bool Mii::TransactionComplete() const {
+    return complete;
+}
+
+ResultCode Mii::GetStatus() const {
+    return ResultSuccess;
+}
+
+void Mii::ExecuteInteractive() {
+    UNREACHABLE_MSG("Unexpected interactive applet data!");
+}
+
+void Mii::Execute() {
+    if (complete) {
+        return;
+    }
+
+    const auto callback = [this](Core::Frontend::MiiParameters parameters) {
```

Comment:
```suggestion
    const auto callback = [this](const Core::Frontend::MiiParameters& parameters) {
```

Otherwise this is copying 88 bytes every time it's called